roles/ceph-client/tasks/create_users_keys.yml

```diff
@@ -43,3 +43,12 @@
   when:
     - cephx
     - keys | length > 0
+
+- name: chmod key(s)
+  file:
+    path: "/etc/ceph/{{ cluster }}.{{ item.name }}.keyring"
+    mode: "{{ item.mode }}"
```

Comment:
you need to edit `roles/ceph-client/defaults/main.yml` and update structure of `keys`